### DISCUSSION TITLE
Changing base resistance without changing estimated power-resistance-cadence relationship

### DISCUSSION THREAD

**Discussion by HilmiiKumdakci** [cagnulein/qdomyos-zwift#260] [Q&A]
Hi,

I am using Android version with BIKE 500 DOMYOS. In 0% slope, it sets resistance into 5. I want to change this into 6 without changing anything else, such as in the same cadence for two versions, I should be generating more power even 0% slope sets resistance into 6th grade.

Is it possible ? If so, how can I do it ?

Thanks

**Comment by cagnulein** [comment]
change the zwift resistance offset to 6 in the QZ settings

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno dom 11 apr 2021 alle ore 17:34 HilmiiKumdakci <
***@***.***> ha scritto:
